docs/marketplace/fastlane/testing-distribution/index.md

```diff
@@ -58,6 +61,8 @@ fastlane add_plugin appcircle_testing_distribution
 - `personalAPIToken`: The Appcircle Personal API token is used to authenticate and secure access to Appcircle services. Add this token to your credentials to enable its use in your pipeline and ensure authorized actions within the platform.
 - `profileName`: Specifies the profile that will be used for uploading the app.
 - `createProfileIfNotExists`: Ensures that a user profile is automatically created if it does not already exist; if the profile name already exists, the app will be uploaded to that existing profile instead.
```

Comment:
This line is not edited in this PR but should be updated. 

> Ensures that a user profile is automatically created
The profile is not a user profile, it is a testing distribution profile.

Reply:
➕ 1

Reply:
@burakberkkeskin It will be better to fix that line in the PR below since it was edited there
- https://github.com/appcircleio/appcircle-docusaurus/pull/763/files

![Screenshot 2024-12-11 at 15 47 34](https://github.com/user-attachments/assets/20db8cd4-b5d1-40f4-840d-d9a118174151)

cc/ @hiilhan